client/src/components/common/ScrollBar.scss

```diff
@@ -8,26 +8,52 @@
 
   &-content {
     width: 100%;
-    padding-bottom: 4px;
+    padding-bottom: 15px;
     margin-bottom: 4px;
     
     &-inner {
       overflow: auto;
+      padding-bottom: 15px;
+      &::-webkit-scrollbar {
```

Comment:
these webkit-specific selectors will only work on chrome, safari - not firefox and others. In the docs it says: ```Non-standard
This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.```